### DISCUSSION TITLE
Suddenly Prime Icons library CDN not working

### DISCUSSION THREAD

**Discussion by bhoomesh-spe** [primefaces/community#1657] [PrimeIcons]
Hello Everyone,

Suddenly, PrimeIcons not working. Below CDN for Prime Icons not working. Please refer to the below screenshots for the same.

https://unpkg.com/primeicons/primeicons.css

We have used the below node modules for PrimeReact, PrimeIcons, and Prime Flex.

"primeflex": "^3.3.1",
"primeicons": "^6.0.1",
"primereact": "^9.6.1",

Please help us to resolve this prime icons issue. When can we expect the above CDN for PrimeIcons to be available?

![321916292-d0d7f30b-530a-4be9-b366-07091a5363ac](https://github.com/primefaces/community/assets/99523338/74197171-c07c-4a21-88d7-d9cb2c7117b3)

Thanks

**Comment by onursenture** [comment]
Hi, Did you try https://unpkg.com/primeicons@7.0.0/primeicons.css? The link you shared is redirecting to the 7.0.0 URL.
